File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/ManifestationDto.java
package com.afpa.cda.dto;

import java.util.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class ManifestationDto {
	private int id;
	private String label;
	private Date dateValidation;
	private AdminDto validateur;
	private AnimationDto animation;
	private Date dateDebut;
	private Date dateFin;
	private int cout;
	private SalleDto salle;
	private int prixBillet;
	private int reservations;
	private int reservationsVip;
	private int rentabilite;
	private AdminDto annulateur;
	private Date dateAnnulation;
	
	
	

}

File: Sources/gestion-culturelle-front/project/src/app/model/manifestationDto.ts
import { SalleDto } from "./salleDto";
import { AnimationDto } from './animationDto';
import { User } from './user';

export class ManifestationDto {
    id: number;
    label: string;
    dateValidation: Date;
    validateur: User;
    animation: AnimationDto;
    dateDebut: Date;
    dateFin: Date;
    cout: number;
    salle: SalleDto;
    prixBillet: number;
    reservations: number;
    reservationsVip: number;

    constructor(id?: number,
        label?: string,
        dateValidation?: Date,
        validateur?: User,
        animation?: AnimationDto,
        dateDebut?: Date,
        dateFin?: Date,
        cout?: number,
        reservations?: number,
        reservationsVip?: number,
        prixBillet?: number,
        salle?: SalleDto,
    ) {
        this.id = id;
        this.label = label;
        this.dateValidation = dateValidation;
        this.validateur = validateur;
        this.animation = animation;
        this.dateDebut = dateDebut;
        this.dateFin = dateFin;
        this.cout = cout;
        this.prixBillet = prixBillet;
        this.reservations = reservations;
        this.reservationsVip = reservationsVip;
        this.salle = salle;

    }
}


